regolith/schemas.py

```diff
@@ -0,0 +1,662 @@
+                                'year': 2017}],
+                        'title': 'Dr.'},
+             'projects': {'_id': 'Cyclus',
+                          'name': 'Cyclus',
+                          'description': 'Agent-Based Nuclear Fuel Cycle Simulator',
+                          'logo': 'http://fuelcycle.org/_static/big_c.png',
+                          'other': [
+                              'Discrete facilities with discrete material transactions',
+                              'Low barrier to entry, rapid payback to adoption'],
+                          'repo': 'https://github.com/cyclus/cyclus/',
+                          'team': [{'begin_month': 'June',
+                                    'begin_year': 2013,
+                                    'end_month': 'July',
+                                    'end_year': 2015,
+                                    'name': 'Anthony Scopatz',
+                                    'position': 'Project Lead'}],
+                          'website': 'http://fuelcycle.org/'},
+             'proposals': {'_id': 'mypropsal',
+                           'ammount': 1000000.0,
+                           'authors': ['Anthony Scopatz',
+                                       'Robert Flanagan'],
+                           'currency': 'USD',
+                           'day': 18,
+                           'durration': 3,
+                           'full': {
+                               'benefit_of_collaboration': 'http://pdf.com/benefit_of_collaboration',
+                               'cv': [
+                                   'http://pdf.com/scopatz-cv',
+                                   'http://pdf.com/flanagan-cv'],
+                               'narrative': 'http://some.com/pdf'},
+                           'month': 'Aug',
+                           'pi': 'Anthony Scopatz',
+                           'pre': {
+                               'benefit_of_collaboration': 'http://pdf.com/benefit_of_collaboration',
+                               'cv': [
+                                   'http://pdf.com/scopatz-cv',
+                                   'http://pdf.com/flanagan-cv'],
+                               'day': 2,
+                               'month': 'Aug',
+                               'narrative': 'http://some.com/pdf',
+                               'year': 1998},
+                           'status': 'submitted',
+                           'title': 'A very fine proposal indeed',
+                           'year': 1999},
+             'students': {'_id': 'Human A. Person',
+                          'aka': ['H. A. Person'],
+                          'email': 'haperson@uni.edu',
+                          'university_id': 'HAP42'}}
+
+SCHEMAS = {'abstracts': {'_description': {'description': 'Abstracts for a conference or workshop. This is generally public information\n\n'},
```

Comment:
many lines too long here, also probably shouldn't have `\n\n` at the end of them. That can be added in the doc generator